modules/sdk/project-templates/project-templates-api/src/test/java/com/liferay/project/templates/simpleapi/ProjectTemplatesApiTest.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.project.templates.simpleapi;
```

Comment:
Source formatter didnt like package names ending in `.api` or `.test` unless it matched the bundle-name, so i changed it to simpleapi. Do you have a better name?